book/src/api-lighthouse.md

```diff
@@ -297,7 +360,10 @@ health of the execution node that the beacon node is connected to.
 - `latest_cached_block_number` & `latest_cached_block_timestamp`: the block
 number and timestamp of the latest block we have in our block cache.
 	- For correct execution client voting this timestamp should be later than the
-`voting_period_start_timestamp`.
+`voting_period_start_timestamp`. 
+
+[comment]: <> (What is `voting_period_start_timestamp`? A search on sigp/lighthouse github only returns the parameter for this .md file and doesn't seem to be defined elsewhere)
+
```

Comment:
```suggestion
`voting_target_timestamp`. 

```

Nice find! The variable name was wrong, we wanted `voting_target_timestamp` instead.